### ISSUE TITLE
drawers not being defined correctly

### ISSUE THREAD

**Issue by benbenlaw** [jaquadro/StorageDrawers#934]
https://github.com/TeamPneumatic/pnc-repressurized/issues/759

**Comment by desht** [comment]
Note: this should be as simple as adding: 
```java
@Override
public boolean allowsMovement(BlockState state, IBlockReader worldIn, BlockPos pos, PathType type) {
      return false;
}
```
to your `BlockDrawers` class.  The issue is that drawers appear to have a non-full but movement-blocking collision shape, but don't override this method.  So entities think they can pathfind through them, when they can't.  This is particularly noticeable with PneumaticCraft drones, which tend to find this problem very frequently!

(You'll notice that many non-solid vanilla blocks also override this method to return false, for this very reason)

**Comment by jaquadro** [comment]
Fixed in the latest 1.16 release
